Gems/Atom/RPI/Code/Source/RPI.Reflect/Image/StreamingImageAsset.cpp

```diff
@@ -93,18 +93,57 @@ namespace AZ
         {
             return m_totalImageDataSize;
         }
-        
+
+        RHI::ImageDescriptor StreamingImageAsset::GetImageDescriptorForMipLevel(AZ::u32 mipLevel) const
+        {
+            RHI::ImageDescriptor imageDescriptor = GetImageDescriptor();
+
+            const ImageMipChainAsset* mipChainAsset = GetImageMipChainAsset(mipLevel);
+            if (mipChainAsset)
+            {
+                auto mipChainIndex = GetMipChainIndex(mipLevel);
+                auto mipChainOffset = aznumeric_cast<AZ::u32>(GetMipLevel(mipChainIndex));
```

Comment:
is mipChainOffset different than mipLevel? Just seems weird to call getIndex on the level to get the index, then call getLevel on the index to get the level again.